CHANGES.md

```diff
@@ -1,3 +1,7 @@
+# 20.12
+
+   * The radiation frequency space advection was converted to C++ (#1189)
```

Comment:
I don't have any problem with this but just FYI my strategy has been to only update this file for user-facing/problem setup-related changes, not for internal implementation details.

Reply:
removed